spec/locale.html

```diff
@@ -353,4 +353,36 @@ <h1>get Intl.Locale.prototype.region</h1>
       </emu-alg>
     </emu-clause>
   </emu-clause>
+
+  <emu-clause id="sec-properties-of-intl-locale-instances">
+    <h1>Properties of Intl.Locale Instances</h1>
+
+    <p>
+      Intl.Locale instances are ordinary objects that inherit properties from %Locale.prototype%.
+    </p>
+
+    <p>
+      Intl.Locale instances have an [[InitializedLocale]] internal slot.
+    </p>
+
+    <p>
+      Intl.Locale instances also have several internal slots that are computed by the constructor:
+    </p>
+
+    <ul>
+      <li>[[Locale]] is a String value with the language tag of the locale whose localization is used for formatting.</li>
+      <li>[[Calendar]] is a String value with the *"type"* given in <a href="https://unicode.org/reports/tr35/#Key_And_Type_Definitions_">Unicode Technical Standard #35 Part 1 Core, Section 3.6.1 Key and Type Definitions</a> for the calendar used for formatting, or is *undefined*.</li>
+      <li>[[Collation]] is a String value with the *"type"* given in <a href="https://unicode.org/reports/tr35/#Key_And_Type_Definitions_">Unicode Technical Standard #35 Part 1 Core, Section 3.6.1 Key and Type Definitions</a> for the collation, or is the String value *"default"*, or is *undefined*.</li>
+      <li>[[HourCycle]] is a String value indicating whether the 12-hour format (*"h11"*, *"h12"*) or the 24-hour format (*"h23"*, *"h24"*) should be used, or is *undefined*.</li>
+      <li>[[NumberingSystem]] is a String value with the *"type"* given in <a href="https://unicode.org/reports/tr35/#Key_And_Type_Definitions_">Unicode Technical Standard #35 Part 1 Core, Section 3.6.1 Key and Type Definitions</a> for the numbering system used for formatting, or is *undefined*.</li>
+    </ul>
+    <p>
+      If the [[RelevantExtensionKeys]] internal slot of %Locale% contains *"kn"* and/or *"kf"*, Intl.Locale instances will also have the following internal slots:
+    </p>
+
+    <ul>
+      <li>[[CaseFirst]] is one of the String values *"upper"*, *"lower"*, or *"false"*, or is *undefined*. This internal slot only exists if the [[RelevantExtensionKeys]] internal slot of %Locale% contains *"kf"*.</li>
```

Comment:
I also just realize [[CaseFirst]] could contains not just *undefined* or String *"upper"*, *"lower"*, or *"false"* but other strings which fit "with the *"type"* given in <a href="https://unicode.org/reports/tr35/#Key_And_Type_Definitions_">Unicode Technical Standard #35 Part 1 Core, Section 3.6.1 Key and Type Definitions</a> "

For example, in the following code:
```
let b = new Intl.Locale("en-u-kf-efg")
```
[[CaseFirst]] is "efg".

Reply:
ah, thank you! looking at it now